tgui/packages/tgui/interfaces/RequestsConsole.tsx

```diff
@@ -0,0 +1,46 @@
+import { useBackend } from '../backend';
```

Comment:
rename this file `index.tsx` inside of RequestsConsole folder. it should function the same

Reply:
Ah, thank, you didn't know that was a thing. 👍

Reply:
of course. extra info: this is a bit of a contentious topic in JS because it can fill your repo (and your editor tabs) with index files. 

however, it's nice for organization because you can import something using `import { SomeComponent } from "./SomeComponent"` and that can either mean a file `SomeComponent.tsx` or a folder `SomeComponent/index.tsx`.